vtex/io-clients
# Adding method to search for documents and getting raw info

#### What is the purpose of this pull request?

It adds a new method in the MD Factory to allow search results that contain pagination info.

#### How should this be manually tested?
There is a beta version released, so you can use it in your application (`2.7.2-beta`). @renanmagagnin tested his project and it's working. You can see the screenshot below:
![image (18)](https://user-images.githubusercontent.com/19495917/111330943-2bc80800-864f-11eb-91fb-592541e3bbdb.png)

#### Types of changes

#### Chores checklist
